swupd/delta.go

```diff
@@ -96,6 +96,31 @@ func createDeltasFromManifests(c *config, oldManifest, newManifest *Manifest, nu
 	return deltas, nil
 }
 
+// deltaLargerThanFull returns true if the delta file is larger than the
```

Comment:
It also returns true if it's the same size :eagle:

Reply:
ogoodpoint